### ISSUE TITLE
Unable to 'make deb' on packages situated on http://releases.ansible.com/ansible

### ISSUE THREAD

**Issue by baddev** [ansible/ansible#8129]
```
##### Bug Report : Unable to 'make deb' on packages situated on http://releases.ansible.com/ansible

##### Ansible versions : at least, 1.6.6, 1.6, 1.6.5 (all 3 tested)

##### Environment : Debian Wheezy 7.5.0

##### Summary : The 'make deb' command rightfully initiated returns systematic error

##### Steps To Reproduce :

    1 - download the archive
    2 - untar it
    3 - cd to it
    4 - run 'make deb' command

##### Expected Results : A fully useful deb package

##### Actual Results : Only relevant lines have been reproduced here

Error log :

cat: VERSION: No such file or directory
fatal: Not a git repository (or any parent up to mount point /home)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
Cleaning up distutils stuff
[...]
reading manifest file 'ansible.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
warning: no files found matching 'packaging/distutils/setup.py'
writing manifest file 'ansible.egg-info/SOURCES.txt'
warning: sdist: standard file not found: should have one of README, README.rst, README.txt

running check
creating ansible-1.6.5
creating ansible-1.6.5/ansible.egg-info
[...]
hard linking packaging/rpm/ansible.spec -> ansible-1.6.5/packaging/rpm
copying setup.cfg -> ansible-1.6.5
Writing ansible-1.6.5/setup.cfg
creating dist
Creating tar archive
removing 'ansible-1.6.5' (and everything under it)
tar: dist/ansible-.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
cp: cannot stat `packaging/debian': No such file or directory
sed: can't read deb-build/unstable/ansible-/debian/changelog: No such file or directory
make: *** [debian] Error 2
```

**Comment by ansibot** [comment]
Thanks for filing a ticket! I am the friendly GitHub Ansibot. I see you did not fill out your issue description based on our new issue template. Please copy the contents of this [template](https://raw2.github.com/ansible/ansible/devel/examples/issues/ISSUE_TEMPLATE.md) and **paste it into the description** of your ticket. 

Our system will automatically close tickets that do not have an issue template filled out within 7 days. Please note that due to large interest in Ansible, humans may not comment on your ticket if you ask them questions. Don't worry, you're still in the queue and the robots are looking after you.

**Comment by mpdehaan** [comment]
While we can look at this, those releases are build output from "make sdist".  If you want to run "make deb", you can definitely checkout the appropriate tag.

cc @jlaska

**Comment by baddev** [comment]
On 07/16/2014 05:23 PM, Michael DeHaan wrote:

> While we can look at this, those releases are build output from "make
> sdist". If you want to run "make deb", you can definitely checkout the
> appropriate tag.

Sorry I don't understand your answer : what do you mean by "checkout the
appropriate tag" ?

Thanks,

## 

“One original thought is worth a thousand mindless quotings.”
“Le vrai n'est pas plus sûr que le probable.”

```
                                          Diogene Laerce
```

**Comment by srgvg** [comment]
@baddev he means doing a git clone of this repo, then a git checkout of the tagged release, and doing make deb from there.

**Comment by baddev** [comment]
On 07/16/2014 06:18 PM, Serge van Ginderachter wrote:

> @baddev https://github.com/baddev he means doing a git clone of this
> repo, then a git checkout of the tagged release, and doing make deb from
> there.

Thanks I didn't know this command.I get this error message :

##################################################################

Cloning into 'ansible'...
remote: Reusing existing pack: 61529, done.
remote: Counting objects: 105, done.
remote: Compressing objects: 100% (99/99), done.
remote: Total 61634 (delta 42), reused 18 (delta 0)
Receiving objects: 100% (61634/61634), 19.32 MiB | 3.71 MiB/s, done.
Resolving deltas: 100% (38421/38421), done.
Note: checking out 'v1.6.6'.

You are in 'detached HEAD' state. You can look around, make experimental
changes and commit them, and you can discard any commits you make in this
state without impacting any branches by performing another checkout.

If you want to create a new branch to retain commits you create, you may
do so (now or later) by using -b with the checkout command again. Example:

  git checkout -b new_branch_name

HEAD is now at 69d85c2... Version bump for release 1.6.6
Cleaning up distutils stuff
rm -rf build
rm -rf dist
Cleaning up byte compiled python stuff
find . -type f -regex "._.py[co]$" -delete
Cleaning up editor backup files
find . -type f ( -name "_~" -or -name "#_" ) -delete
find . -type f ( -name "_.swp" ) -delete
Cleaning up manpage stuff
find ./docs/man -type f -name "_.xml" -delete
find ./docs/man -type f -name "_.asciidoc" -delete
find ./docs/man/man3 -type f -name "_.3" -delete
Cleaning up output from test runs
rm -rf test/test_data
Cleaning up RPM building stuff
rm -rf MANIFEST rpm-build
Cleaning up Debian building stuff
rm -rf debian
rm -rf deb-build
rm -rf docs/json
rm -rf docs/js
Cleaning up authors file
rm -f AUTHORS.TXT
python setup.py sdist
  File "setup.py", line 12
    print "Ansible now needs setuptools in order to build. " + \
                                                           ^
SyntaxError: invalid syntax
make: *_\* [sdist] Error 1

##################################################################

## 

“One original thought is worth a thousand mindless quotings.”
“Le vrai n'est pas plus sûr que le probable.”

```
                                          Diogene Laerce
```

**Comment by jlaska** [comment]
There is a small bug in the setup.py that's causing it to fail while displaying an error message.  I pushed a fix for that issue.  Basically, it's telling you that you need to have `setuptools` installed.

**Comment by baddev** [comment]
On 07/17/2014 02:59 PM, James Laska wrote:

> There is a small bug in the setup.py that's causing it to fail while
> displaying an error message. I pushed a fix for that issue. Basically,
> it's telling you that you need to have |setuptools| installed.

No problem, we can wait of this issue. There are plenty of fallback
solution for.

Thanks.

## 

“One original thought is worth a thousand mindless quotings.”
“Le vrai n'est pas plus sûr que le probable.”

```
                                          Diogene Laerce
```

**Comment by ajatt5** [comment]
@jlaska I have the same issue; the setup.py tells me I need to install setuptools but the package is already installed on my server.

**Comment by jimi-c** [comment]
# Update Required

Hi, our apologies on taking so long to follow-up on this issue.

This bug was reported on a version of Ansible that is very out of date, and we're not sure if it's valid anymore. As such, could you please let us know if this issue still applies to the devel branch, or the most recent version of Ansible (currently 2.1.x or the latest commit from the stable-2.1 branch)?

If we do not see any follow-up on this issue, we will close and lock this issue and any further reports should be done in a new issue.

Thanks!

**Comment by ansibot** [comment]
@baddev ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@baddev ping, this issue is still waiting on your feedback. We will close the issue if you do not respond.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@baddev closure has been requested for this issue or it has timed out waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)
